ecosystem/typescript/sdk_v2/src/bcs/move-types/primitives.ts

```diff
@@ -0,0 +1,137 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  MAX_U128_BIG_INT,
+  MAX_U16_NUMBER,
+  MAX_U32_NUMBER,
+  MAX_U64_BIG_INT,
+  MAX_U8_NUMBER,
+  MAX_U256_BIG_INT,
+} from "../consts";
+import { AnyNumber, Uint16, Uint32, Uint8 } from "../../types";
+import { Deserializer } from "../deserializer";
+import { Serializable, Serializer, validateNumberInRange } from "../serializer";
+
+export class Bool extends Serializable {
+  public readonly value: boolean;
+
+  constructor(value: boolean) {
+    super();
+    this.value = value;
+  }
+
+  serialize(serializer: Serializer): void {
+    serializer.serializeBool(this.value);
+  }
+
+  static deserialize(deserializer: Deserializer): Bool {
+    return new Bool(deserializer.deserializeBool());
+  }
+}
+export class U8 extends Serializable {
```

Comment:
nit
```suggestion

export class U8 extends Serializable {
```

Reply:
surprised these got through the formatter